README.md

```diff
@@ -43,3 +52,7 @@ Maintainers
 -------
 * Currently maintained by Mark Carpenter <mark@bird-mtn.dev>
 * Originally written by Maik Schreiber <blizzy@blizzy.de>
+
+Contributing
+-------
+Want to help develop Ebiten UI? Check out our [current issues](https://github.com/ebitenui/ebitenui/issues). Want to know the steps on how to start contributing, take a look at the [open source guide](https://opensource.guide/).
```

Comment:
What do you think about updating the link for the open source guide to : https://opensource.guide/how-to-contribute/